cblox/include/cblox/core/submap_collection_inl.h

```diff
@@ -0,0 +1,376 @@
+//
```

Comment:
Let's be consistent on our headers. Remove this. I'll add you to the contributors list (and make one).

Reply:
These headers are auto generated by clion. Feel free to remove any of them.
Sounds great :)